src/main/java/edu/kis/powp/jobs2d/commands/SetPositionCommand.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.commands;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class SetPositionCommand implements DriverCommand {
+
+    private final int x;
+    private final int y;
+    private final Job2dDriver driver;
+
+    public SetPositionCommand(int posX, int posY, Job2dDriver driver) {
```

Comment:
The solution lacks flexibility, using different drivers with commands is difficult (e.g., you could consider setter).